packages/cfd/src/Constants/mt5-migration-modal-content.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Localize } from '@deriv/translations';
+
+const getMigrationModalDetails = (to_account: string) => {
```

Comment:
`const getMigrationModalDetails = (to_account: string) => ( [.. the array ...] )`?